src/server/services/post.service.ts

```diff
@@ -114,53 +158,65 @@ export const getPostsInfinite = async ({
       return { items: [] };
     }
 
-    const collectionItemModelsAND: Prisma.Enumerable<Prisma.CollectionItemWhereInput> =
-      getAvailableCollectionItemsFilterForUser({ permissions, user });
-
-    AND.push({
-      collectionItems: {
-        some: {
-          collectionId,
-          AND: collectionItemModelsAND,
-        },
-      },
-    });
+    AND.push(Prisma.sql`p."collectionId" = ${collectionId}`);
```

Comment:
I don't think you should filter by this value other than for the single purpose of excluding this post from the Infinite query. This doesn't mean a post belong to a collection (Or rather, doesn't mean it's been "collected") - this means the post was created **for** that collection and all images uploaded to it will in turn be linked to the collection. 

@manuelurenah  can correct me there but I believe that was the logic you guys discussed when chatting about drag-and-dropping images.

Reply:
Keeping this in case at some point we want to get the posts belonging to a collection